cmd/vic-machine/create/create.go

```diff
@@ -998,6 +1004,16 @@ func (c *Create) processRegistries() error {
 		c.InsecureRegistries = append(c.InsecureRegistries, *regurl)
 	}
 
+	// load a list of whitelisted registries
+	for _, registry := range c.whitelistRegistries {
+		regurl, err := validate.ParseURL(registry)
```

Comment:
They're currently stored as url.  Insecure registries are also stored as url.  If I change it, then I am in vic-machine upgrade territory and that seems more trouble.